src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,184 @@
+            }
+        }
+
+        /// <summary>
+        /// Title of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Title
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Original Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string OriginalUrl
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_original_url_get(_item_handle);
+            }
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Back Forward list of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardList
+    {
+        private IntPtr _list_handle;
+
+        /// <summary>
+        /// Creates a Back Forward List object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal BackForwardList(IntPtr handle)
+        {
+            _list_handle = handle;
+        }
+
+        /// <summary>
+        /// Current item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem CurrentItem
+        {
+            get
+            {
+                IntPtr itemPtr = Interop.ChromiumEwk.ewk_back_forward_list_current_item_get(_list_handle);
+                if(itemPtr != null) {
+                    BackForwardListItem item = new BackForwardListItem(itemPtr);
+                    return item;
+                }
+                else
+                   return null;
+            }
+        }
+
+        /// <summary>
+        /// Previous item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem PreviousItem
+        {
+            get
+            {
+                IntPtr itemPtr = Interop.ChromiumEwk.ewk_back_forward_list_previous_item_get(_list_handle);
+                if(itemPtr != null) {
+                    BackForwardListItem item = new BackForwardListItem(itemPtr);
+                    return item;
+                }
+                else
+                    return null;
+            }
+        }
+
+        /// <summary>
+        /// Gets the back forward list count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public uint Count
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_count(_list_handle);
+            }
+        }
+        /// <summary>
+        /// Gets the list containing the items preceding the current item
+        /// limited by limit.
+        /// </summary>
+        /// <param name="limit"> limit The number of items to retrieve.</param>
```

Comment:
How to set  as unlimited?

Reply:
According to below implementation if limit is set to -1, it will be unlimited
Eina_List* ewk_back_forward_list_n_back_items_copy(const Ewk_Back_Forward_List* list, int limit)
{
  EINA_SAFETY_ON_NULL_RETURN_VAL(list, NULL);
  /* LCOV_EXCL_START */
  int count = list->GetBackListLength();
  if (limit > -1 && limit < count) {
    count = limit;
  }

Reply:
please update description

Reply:
Done